skimage/segmentation/active_contour_model.py

```diff
@@ -54,9 +54,9 @@ def active_contour(image, snake, alpha=0.01, beta=0.1,
         be combined by parsing 'fixed-free', 'free-fixed'. Parsing
         'fixed-fixed' or 'free-free' yields same behaviour as 'fixed' and
         'free', respectively.
-    coordinates : {'rc' or 'xy'}, optional
-        Whether to use rc or xy coordinates. The 'xy' option (current default)
-        will be removed in version 0.18.
+    coordinates : {'rc'}, optional
+        This option remains for compatibility purpose only and has no effect.
```

Comment:
```suggestion
        This option remains for compatibility purpose only and has no effect.
        It was introduced in 0.16 with the ``'xy'`` option, but since 0.18, 
        only the ``'rc'`` option is valid.
```

Reply:
A little bit of history goes a long way to future us!